decidim-admin/app/views/decidim/admin/officializations/index.html.erb

```diff
@@ -55,13 +55,14 @@
         <tbody>
           <% @users.each do |user| %>
             <tr data-user-id="<%= user.id %>">
-              <td><%= user.name %></td>
-              <td><%= user.nickname %></td>
+              <td><%= link_to user.name, profile_path(user) %></td>
+              <td><%= link_to user.nickname, profile_path(user) %></td>
```

Comment:
We should probably choose the implicit (`profile_path(user)`) or the explicit (`profile_path(user.nickname)`) version and use it consistenly. Maybe the implicit is currently more common?

Reply:
I suggest implicit as we can change for /profiles/:id if we need it without changing all view.

Reply:
Agreed, would be a good rubocop cop :smiley: